test/functional/driver/convert-to-entity/convert-to-entity.ts

```diff
@@ -0,0 +1,30 @@
+import "reflect-metadata";
+import { expect } from "chai";
+import { createTestingConnections, closeTestingConnections, reloadTestingDatabases } from "../../../utils/test-utils";
+import { Connection } from "../../../../src/connection/Connection";
+import { Post } from "./entity/Post";
+
+describe("driver > convert raw results to entity", () => {
+
+    let connections: Connection[];
+    before(async () => connections = await createTestingConnections({
+        entities: [Post],
+        schemaCreate: true,
+        dropSchemaOnConnection: true
+    }));
+    beforeEach(() => reloadTestingDatabases(connections));
+    after(() => closeTestingConnections(connections));
+
+    it.only("should return null value in entity property when record column is null", () => Promise.all(connections.map(async connection => {
+        const userRepository = connection.getRepository<Post>("Post");
```

Comment:
also no need to do `connection.getRepository<Post>("Post")` you can simply do `connection.getRepository(Post)`